### ISSUE TITLE
[FEATURE]

### ISSUE THREAD

**Issue by inpharmaticist** [vitorpamplona/amethyst#687]
I put up a [bounty](https://bountsr.org/meetup-replacement/) on bountsr for a meetup replacement and someone put a very cool website together called [Flockstr](https://www.flockstr.com/explore). I would love to be able to see and interact with these events on amethyst. I think they are a special kind currently that doesn't show in app but thought I'd ask because it seems possible (I'm not a dev).

[Here's](https://www.flockstr.com/event/naddr1qqyxxenxxdjnwcfcqyxhwumn8ghj7mn0wvhxcmmvqgsv83cj9336755plm3mjvudzzhxtvtxlln0w0cuyuk9cyfqvpmnr4crqsqqql9ny0fr8u) an event/calendar I created for my meetup.

I'll offer 100k sats to be able to at least
* RSVP to events
* Follow calendars
* Post to the discussion section of an event
* Get notifications for new events on a calendar

Some nice-to-haves that may be worth an additional bounty
* Post events and generally manage a calendar
* Set reminder notifications for upcoming events

**Comment by anthony-robin** [comment]
I would love te be able to visualize NIP-52 kinds `31922` and `31923` directly inside Amethyst on my feed :)

**Comment by Teodor82** [comment]
Hi! I’m interested in this bounty.
Before starting, could you clarify:
– What is the minimum scope required for the bounty
– whether a USD payment is possible instead of sats
– how “done” is verified

Thanks!

**Comment by inpharmaticist** [comment]
@Teodor82 I dont think flockstr is being maintained anymore. So without a means to create these events, I dont think this bounty is currently possible.

**Comment by vitorpamplona** [comment]
done! I will be released in the next version

**Comment by inpharmaticist** [comment]
Looking forward to testing!

**Comment by inpharmaticist** [comment]
Bounty paid. Zapped the amethyst profile.
